### PR TITLE
refactor(storage): rename volume_id -> volume_name_ref

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: pkg/middleend/qos.go [hunk 1]
```diff
@@ -249,9 +249,6 @@ func (s *Server) verifyQosVolume(volume *pb.QosVolume) error {
 	if err := resourcename.Validate(volume.Name); err != nil {
 		return err
 	}
-	if volume.VolumeId == nil || volume.VolumeId.Value == "" {
```
Comment (glimchb) [inline pkg/middleend/qos.go:252]: @artek-koltun please note this removal

**Comment by glimchb** [comment]
depends on https://github.com/opiproject/godpu/pull/252

File: pkg/middleend/qos.go [hunk 2]
```diff
@@ -249,9 +249,6 @@ func (s *Server) verifyQosVolume(volume *pb.QosVolume) error {
 	if err := resourcename.Validate(volume.Name); err != nil {
 		return err
 	}
```
Comment (glimchb) [inline pkg/middleend/qos.go:251]: @artek-koltun please note this removal

File: pkg/middleend/qos.go [hunk 3]
```diff
@@ -249,9 +249,6 @@ func (s *Server) verifyQosVolume(volume *pb.QosVolume) error {
...
-	if volume.VolumeId == nil || volume.VolumeId.Value == "" {
-		return fmt.Errorf("volume_id cannot be empty")
-	}
 
 	if volume.MinLimit != nil {
```
Comment (glimchb) [inline pkg/middleend/qos.go:253]: @artek-koltun please note this removal

**Review by artek-koltun — APPROVED** [review]

**Pull request merged**